src/wp-includes/formatting.php

```diff
@@ -1597,6 +1598,11 @@ function remove_accents( $string, $locale = '' ) {
 	}
 
 	if ( seems_utf8( $string ) ) {
```

Comment:
Thinking an inline comment would probably be useful here too. Perhaps something like (copied from the tests):

```
// Unicode sequence normalization from NFD (Normalization Form Decomposed)
// to NFC (Normalization Form [Pre]Composed), the encoding used in this function.
```

Reply:
Good suggestion 👍🏻 Updated!